File: py/taller.py/30.py
# Credenciales del sistema
USER_SYS = "admin"
PASS_SYS = "1234"
ROL_SYS = "admin"

user_input = input("Usuario: ")
pass_input = input("Contraseña: ")
rol_input = input("Rol: ")

# Debe cumplir las tres estrictamente
if user_input == USER_SYS and pass_input == PASS_SYS and rol_input == ROL_SYS:
    print("Acceso concedido. Bienvenido Administrador.")
else:
    print("Acceso denegado. Uno o más datos son incorrectos.")

File: js/taller.js/30.js
const USER_SYS = "admin";
const PASS_SYS = "1234";
const ROL_SYS = "admin";

let userIn = prompt("Usuario:");
let passIn = prompt("Contraseña:");
let rolIn = prompt("Rol:");

if (userIn === USER_SYS && passIn === PASS_SYS && rolIn === ROL_SYS) {
    console.log("Acceso concedido. Bienvenido Administrador.");
} else {
    console.log("Acceso denegado. Uno o más datos son incorrectos.");
}